### PR TITLE
Add usecase for find in dg

### LINKED ISSUE #190: Add in DG usecase for find

### PR DESCRIPTION
**Pull request by Feng1231**
Update error in referring to list

Add new use case for find
closes #190

### CODE REVIEW

**Comment by Feng1231** [comment]
Okay but extensions looks weird with both bullet points and the ordered number list. But without the bullet point, idk how else to indent it. Any one can help? 
image

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -396,12 +396,38 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+   Use case ends.
+
+**Extensions**
+
+* 1a. System detects error in entered command.
+    * 1a1. System displays error message and does not clear command line.
+    * 1a2. User enters new command.
+
+* Steps 1a1-1a2 are repeated until all details entered are correct.
+* Use cases resumes from step 2.
```
Comment (TY1Fan) [inline docs/DeveloperGuide.md:422]: I think it should be use case resumes instead of use cases

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -396,12 +396,38 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+
+* 1b. System detects error in parameters.
+    * 1b1. System displays error message and does not clear command line.
+    * 1b2. User enters command with correct parameters.
+* Steps 1b1-1b2 are repeated until all details entered are correct.
+* Use cases resumes from step 2.
```
Comment (TY1Fan) [inline docs/DeveloperGuide.md:429]: Same for this line

Comment (Feng1231) [reply docs/DeveloperGuide.md:429]: Thats a good point, I copied from UC01. Should we just remove this?

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Feng1231) [reply docs/DeveloperGuide.md:422]: Thanks

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (TY1Fan) [reply docs/DeveloperGuide.md]: @teamotitan @Soumyadip-cmd @DennieDan  what do yall think?

Comment (Soumyadip-cmd) [reply docs/DeveloperGuide.md]: I agree with Yifan.

**Pull request merged**